pkg/providers/azure.go

```diff
@@ -85,38 +89,36 @@ func InitiateAzureOIDCFlow(ctx context.Context, sc *SetUpCmd, s spinner.Spinner)
 	return nil
 }
 
-func (sc *SetUpCmd) createAzApp() error {
+func (sc *SetUpCmd) createAzApp(ctx context.Context) error {
 	log.Debug("Commencing Azure app creation...")
 	start := time.Now()
 	log.Debug(start)
 
 	createApp := func() error {
-		createAppCmd := exec.Command("az", "ad", "app", "create", "--only-show-errors", "--display-name", sc.AppName)
-
-		out, err := createAppCmd.CombinedOutput()
+		azCred, err := cred.GetCred()
 		if err != nil {
-			log.Printf("%s\n", out)
-			return err
+			return fmt.Errorf("getting credentials: %w", err)
 		}
+		graphClient, _ := msgraph.NewGraphServiceClientWithCredentials(azCred, []string{cloud.AzurePublic.Services[cloud.ResourceManager].Endpoint + "/.default"})
```

Comment:
I couldn't use the existing graphClient because of the way it's implemented using graphServiceClient and here I just want the graphClient with default setting.